oxide_math/helpers/rect_size_helpers.rs

```diff
@@ -0,0 +1,10 @@
+use crate::components::rect::*;
+use legion::*;
+
+pub fn set_rect_size(world: &mut World, entity: &Entity, width: f32, height: f32) {
```

Comment:
```suggestion
pub fn add_rect_with_size(world: &mut World, entity: &Entity, width: f32, height: f32) {
```
looks more appropriate

Reply:
This function doesn't add Rect component, just sets width and height. So I think current code is more appropriate

Reply:
```rust
let rect_entity: Entity = world.push((Position {
    x: 1.0f32,
    y: 1.0f32,
},));
set_rect_size(&mut world, &rect_entity, 2.0f32, 3.0f32);
```

this test case looks like it adds rect compontent with specific size.

Reply:
That test case adds rect component with push function, and then re-sets width and height to 2.0 and 3.0 each. set_rect_size function is actually intended for entities which already have Rect component, so its purpose is not for adding components

Reply:
```rust
world.push((Position {
    x: 1.0f32,
    y: 1.0f32,
},));
```
It seems that only Position has been pushed.

Reply:
But in fact, because there isn't any code to check if the Rect component exists, set_rect_size function might add Rect component if entity argument doesn't have Rect component. I'll add some code related to that soon.

Reply:
https://docs.rs/legion/latest/legion/world/struct.Entry.html#method.add_component

According to legion docs, add_component adds a component.

Reply:
But it just replaces the value if the component already exists, so I think 'add' is also not an appropriate name.

How about removing first test case, and add some code in set_rect_size function about checking if Rect component exists?

Reply:
Talking about the name of the function is based on the assumption that you are not changing the implementation, and it is possible to change the implementation.